cfme/tests/v2v/test_v2v.py

```diff
@@ -1,9 +1,97 @@
-"""Test to validate basic navigations.Later to be replaced with End-to-End functional testing."""
 import pytest
+import random
 
+from cfme.automate.import_export import AutomateGitRepository
+from cfme.configure.configuration.region_settings import Tag, Category
+from cfme.infrastructure import host
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
 from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils import conf, testgen
+from cfme.utils.generators import random_vm_name
+from cfme.utils.log import logger
+from cfme.utils.update import update
 
-pytestmark = [pytest.mark.ignore_stream('5.8', '5.9')]
+pytestmark = [
+    pytest.mark.ignore_stream('5.8', '5.9'),
+    pytest.mark.meta(server_roles=['+git_owner'])
+]
+
+
+def pytest_generate_tests(metafunc):
+    argnames1, argvalues1, idlist1 = testgen.providers_by_class(metafunc, [VMwareProvider])
+    argnames2, argvalues2, idlist2 = testgen.providers_by_class(metafunc, [RHEVMProvider])
+
+    new_idlist = []
+    new_argvalues = []
+    new_argnames = ['nvc_prov', 'rhvm_prov']
+
+    for index1, argvalue_tuple1 in enumerate(argvalues1):
+        for index2, argvalue_tuple2 in enumerate(argvalues2):
+            new_idlist.append('{}-{}'.format(idlist1[index1], idlist2[index2]))
+            new_argvalues.append((argvalue_tuple1[0], argvalue_tuple2[0]))
+    testgen.parametrize(metafunc, new_argnames, new_argvalues, ids=new_idlist, scope="module")
+
+
+@pytest.fixture(scope='module')
+def initial_setup(appliance, request, nvc_prov, rhvm_prov):
+    """
+    Steps:
+        1.Validate host credentials
+        2.Assign tags to conversation host
+        3.Import automate domain via github repo
+    """
```

Comment:
**REQUIRED**
I would prefer to split this fixture into three separate functions.

Reply:
@Yadnyawalkya do not import GitHub Repo in Automate Domain, it is now part of Nightly and no need for external import.